mingw-w64-python-fastexcel/PKGBUILD

```
@@ -0,0 +1,48 @@
+# Maintainer: Konstantin Podsvirov <konstantin@podsvirov.su>
+
+_realname=fastexcel
+pkgbase=mingw-w64-python-${_realname}
+pkgname=("${MINGW_PACKAGE_PREFIX}-python-${_realname}")
+pkgver=0.18.0
+pkgrel=1
+pkgdesc="A fast excel file reader for Python, written in Rust (mingw-w64)"
+arch=('any')
+mingw_arch=('mingw64' 'ucrt64' 'clang64' 'clangarm64')
+url='http://fastexcel.toucantoco.dev/'
+msys2_repository_url='https://github.com/ToucanToco/fastexcel'
+license=('spdx:MIT')
+depends=("${MINGW_PACKAGE_PREFIX}-python")
+makedepends=(
+  "${MINGW_PACKAGE_PREFIX}-rust"
+  "${MINGW_PACKAGE_PREFIX}-maturin"
```

```suggestion
  "${MINGW_PACKAGE_PREFIX}-python-maturin"
```
so python-build will work